common/scala/src/main/scala/whisk/core/entity/InvokerInstanceId.scala

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
This file should be called `InstanceId` since it includes multiple implementations.